.github/workflows/android_x64.yml

```
@@ -0,0 +1,168 @@
+name: Android x64
+on:
+  workflow_dispatch:
+  pull_request:
+  merge_group:
+  push:
+    branches:
+      # - master
+      - 'releases/**'
+
+concurrency:
+  # github.ref is not unique in post-commit
+  group: ${{ github.event_name == 'push' && github.run_id || github.ref }}-android-x64-vcpkg
```

```suggestion
  group: ${{ github.event_name == 'push' && github.run_id || github.ref }}-android-x64
```

let's also clean up `vcpkg` from this pipeline, as you have removed its usage.